features/attach_validtoken.feature

```diff
@@ -106,8 +106,8 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         \*Your UA Infra: ESM subscription has EXPIRED\*
         Enabling UA Infra: ESM service would provide security updates for following
         packages:
-          .*libkrad.*
-        \d+ esm-infra security updates NOT APPLIED. Renew your UA services at
+          libkrad0
+        1 esm-infra security update NOT APPLIED. Renew your UA services at
```

Comment:
Based on below change I think this will need to be
```suggestion
        1 esm-infra security update(s) NOT APPLIED. Renew your UA services at
```

Reply:
Good fixed two cases in features where that text would change.